packages/main/src/cache/persist.ts

```diff
@@ -253,28 +251,13 @@ export function persistTokens(syncToken: string, accountId: ID) {
   )
 }
 
-export function persistContacts(
+export function persistContact(
   accountId: ID,
-  connections: people_v1.Schema$Person[]
+  contact: { name?: string; mailbox: string; host: string }
 ) {
-  connections.forEach(connection => {
-    connection.emailAddresses &&
-      connection.emailAddresses.forEach(email => {
```

Comment:
I would rather keep the `cache/` modules focused on moving data into and out of the database, and keep parsing and conversion logic in other places.